test/Microsoft.ML.TestFramework/DataPipe/TestDataPipeBase.cs

```diff
@@ -438,7 +438,7 @@ protected void VerifyArgParsing(IHostEnvironment env, string[] strs)
 
             // Note that we don't pass in "args", but pass in a default args so we test
             // the auto-schema parsing.
-            var loadedData = TextLoader.ReadFile(env, new TextLoader.Arguments(), new MultiFileSource(pathData));
+            var loadedData = TextLoader.ReadFile(env, new MultiFileSource(pathData), new TextLoader.Arguments());
```

Comment:
>ReadFile [](start = 40, length = 8)

ReadTextFile?

Reply:
Isn't `text` implied by the name of `TextLoader`?